Tasks/FileTransform/task.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "8ce97e91-56cc-4743-bfab-9a9315be5f27",
+    "name": "FileTransform",
+    "friendlyName": "File Transform",
+    "description": "File Transform",
+    "author": "Microsoft Corporation",
+    "helpMarkDown": "File Transforms & Variable Substitution",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "preview": "true",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "demands": [],
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "instanceNameFormat": "File Transform: $(Package)",
+    "inputs": [
+        {
+            "name": "folderPath",
+            "type": "filePath",
+            "label": "Folder Path",
+            "defaultValue": "$(System.DefaultWorkingDirectory)/**/*.zip",
```

Comment:
defaultValue is zip?

Reply:
🙀

Reply:
Yes we will support zip as well and in-line with deploy task zip makes more sense